meeteval/io/__main__.py

```diff
@@ -0,0 +1,159 @@
+import os
+import sys
+from stat import S_ISFIFO
+from pathlib import Path
+import argparse
+import meeteval
+
+
+def _is_piping_in():
+    """
+    Checks whether input is being piped into the command.
+    """
+    return S_ISFIFO(os.fstat(0).st_mode)
+
+
+def convert(input_files, output_file, input_format, output_format, parser, **kwargs):
+    """
+    Converts one or multiple `input_files` from `input_format` to 
+    `output_format` and writes the result to `output_file`. All input files must 
+    have the same format. The `kwargs` are used to fill missing keys in the 
+    destination format. If a value in `kwargs` is a string, it is formatted with
+    the segment information and the file stem of the input file in the key 
+    `'filestem'`.
```

Comment:
Suggestion:
```
Converts one or multiple `input_files` from `input_format` ('.rttm', '.stm', '.ctm', '.seglst') to 
`output_format` ('.rttm', '.stm', '.seglst') and writes the result to `output_file`.

The main purpose of `kwargs` is to add missing keys from the
format conversion, e.g. '.stm' has speaker, while '.ctm' not.
In case of '.seglst', `kwargs` is used to update every segment.
If a value in `kwargs` is a string, it is formatted with
the segment information and the file stem of the input file in the key 
`'filestem'`.
```